mon-pix/translations/en.json

```
@@ -1221,16 +1221,16 @@
         "direction": "Select several answers."
       },
       "qcu": {
-        "direction": "Select one answer."
+        "direction": "Select a single answer."
       },
       "qrocm": {
-        "direction": "Fill in the empty fields."
+        "direction": "Fill in the {count, plural, =1 {blank} other {blanks}}."
       },
       "recap": {
         "title": "Congratulations! Module completed",
         "backToModuleDetails": "Back to module details",
         "goToForm": "Répondre au questionnaire",
```

on a oublié celui la 
```suggestion
        "goToForm": "Answer the questionnaire",
```